common/lib/xmodule/xmodule/split_test_module.py

```diff
@@ -58,7 +58,7 @@ def build_partition_values(all_user_partitions, selected_user_partition):
 
     display_name = String(
         display_name=_("Display Name"),
-        help=_("This name is used for organizing your course content, but is not shown to students."),
+        help=_("The display name for this component. (Not shown to students)"),
```

Comment:
@yro While you're at it, can you please replace "students" -> "learners" in the string? Thanks!